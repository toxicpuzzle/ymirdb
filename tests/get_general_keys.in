SET bro 0 0 0 1 2
SET cool 1 0 0 0 2
PUSH cool bro
GET bro
GET cool
bye