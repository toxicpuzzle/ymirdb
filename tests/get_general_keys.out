> ok

> ok

> ok

> [0 0 0 1 2]

> [bro 1 0 0 0 2]

> bye
